src/main/resources/application.yml

```diff
@@ -148,7 +148,7 @@ reports:
       page-size: 40
   locations:
     reconciliation:
-      page-size: 40
+      page-size: 20
```

Comment:
It turns out most of the other reconciliations already use 20 🤭 
This should decrease the load on the locations-api